sympy/integrals/integrals.py

```diff
@@ -497,10 +497,8 @@ def try_meijerg(function, xab):
                     antideriv = None
                 else:
                     antideriv = self._eval_integral(
-                        function, xab[0],
-                        meijerg=meijerg1, risch=risch, manual=manual,
-                        conds=conds)
-                    if antideriv is None and meijerg1 is True:
+                        function, xab[0], **eval_kwargs)
+                    if antideriv is None and meijerg is True:
```

Comment:
changed meijerg1 to meigerg; the only way meijerg1 could be True is if meijerg were True because the intervening code only sets meijerg1 to False if the integration failed.